index.html

```diff
@@ -0,0 +1,130 @@
+                                class="menu" id="speaker">Speakers</a></li>
+                        <li class="menu"><a href="#partner"
+                                class="menu" id="sponsor">Sponsors</a></li>
+                        <li class="menu"><a href="about.html"
+                                class="menu" id="openAbout">About</a></li>
+                    </ul>
+
+                </div>
+                <ul class="second-one">
+                    <li> <a href="#main-program" class="link">Programs</a></li>
+                    <li> <a href="#speaker" class="link">Speakers</a></li>
+                    <li> <a href="#partner" class="link">Partner</a></li>
+                    <li> <a href="about.html" class="link-1">About</a></li>
+                </ul>
+            </nav>
+        </header>
+        <!--start of home page-->
+        <section class="heading" id="home-1">
+            <div class="home">
+                <img src="Images\hamburger.png" alt="Humbergar
+                    icon" class="hamburger" width="50px" id="hamburger">
+                <p class="hello">"Hello world"</p><br>
+                <h1 class="sub-title">The Best Online School 2022</h1><br>
+                <P class="description">Microverse is a global school for remote
+                    software developers where students learn through remote pair
+                    programming and don't pay anything until they get a job</P><br>
+                <h4 class="date">2022.6.22 (MON) ~ (Friday)</h4><br>
+                <p class="location">San Francisco, California, United States</p><br><br>
+            </div>
+        </section>
+        <section id="main-pro">
+            <h2 class="sections-title" id="main-program">Main Program</h2><br>
+            <br><br>
+            <div class="sections">
+                <div class="sub-section">
+                    <img class="sub-image" src="Images/html.png" alt=""
+                        id="logo-section">
+                    <h4 class="section-title">HTML</h4>
+                    <p class="sub-description">HTML is the standard markup
+                        language for creating Web pages.</p><br>
+                </div>
+                <div class="sub-section">
+                    <img class="sub-image" src="Images/github.png"
+                        alt="" id="logo-section">
+                    <h4 class="section-title">Github</h4>
+                    <p class="sub-description">A website to host your
+                        repositories online.</p>
+                </div>
+                <div class="sub-section">
+                    <img class="sub-image" src="Images/css.png" alt=""
+                        id="logo-section">
+                    <h4 class="section-title">CSS</h4>
+                    <p class="sub-description">CSS is a language that describes
+                        the style of an HTML document.</p><br>
+                </div>
```

Comment:
- [ ] Kindly make sure to add 2 more cards as suggested in the attached loom video. 
- [ ] Kindly make sure to follow the spacing as suggested in the attached loom video.